set.mm

```diff
@@ -39333,6 +39333,20 @@ technically classes despite morally (and provably) being sets, like ` 1 `
       ACDJUEUGUFUIUGUEADCIKUFBCDHUHBCDRBDCILMNO $.
   $}
 
+  ${
+    $( The class abstraction with a conjunction is a subset of the class
+       abstraction with the left conjunct only.  (Contributed by AV,
+       7-Aug-2024.) $)
+    abanssl $p |- { f | ( ph /\ ps ) } C_ { f | ph } $=
+      ( wa cab cin inab inss1 eqsstrri ) ABDCEACEZBCEZFJABCGJKHI $.
+
+    $( The class abstraction with a conjunction is a subset of the class
+       abstraction with the right conjunct only.  (Contributed by AV,
+       7-Aug-2024.) $)
+    abanssr $p |- { f | ( ph /\ ps ) } C_ { f | ps } $=
+      ( wa cab cin inab inss2 eqsstrri ) ABDCEACEZBCEZFKABCGJKHI $.
+  $}
```

Comment:
can be shortened: simpl, simpr + ss2abi

f1osetex can be shortened similarly: f1of1 + ss2abi

Reply:
the credit is incorrect (--> SN)

Reply:
done